### ISSUE TITLE
Accessibility rating: Button

### ISSUE THREAD

**Issue by tobiasljungstrom** [SKF/ferris#221]
Asses the accessibility of the Button component and put the rating in the component table.

The component should meet the following criteria:

- [x] Tab-able with a clear focus state
- [x] Screen reader should be able to traverse it (if applicable)
- [x] Custom control should work with screen reader
- [x] Focus should be directed to any new items that pop up on the screen
- [x] Color and contrast should be good
